full-stack-tests/rpc-interface/src/common/SideChannels.ts

```diff
@@ -2,31 +2,17 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-import { AgentAuthorizationClient, AgentAuthorizationClientConfiguration } from "@bentley/backend-itwin-client";
-import { AccessToken } from "@itwin/core-bentley";
 import { executeBackendCallback, registerBackendCallback } from "@itwin/certa/lib/utils/CallbackUtils";
 
 const getEnvCallbackName = "getEnv";
-const getClientAccessTokenCallbackName = "getClientAccessToken";
 
 export function exposeBackendCallbacks() {
   registerBackendCallback(getEnvCallbackName, () => {
     return JSON.stringify(process.env);
   });
-
-  registerBackendCallback(getClientAccessTokenCallbackName, async (clientConfiguration: AgentAuthorizationClientConfiguration) => {
-    const authClient = new AgentAuthorizationClient(clientConfiguration);
-    const token = await authClient.getAccessToken();
-    return token ?? "";
-  });
-
```

Comment:
@calebmshafer Any other recommendations for removing AgentAuth out of here?

Reply:
Hm okay, this will need the new auth clients published. Lets get a preliminary version of them out today and test it out that way.

Reply:
Update: There are some issues with pulling service auth into this spot that we are trying to fix but with the release pipeline broken of iTwin.js Core we're stuck at the moment.

Until I fix the release pipeline for a new RC, we'll comment these tests out to not hold up these changes.